lang/LangPrimSource/PyrArrayPrimitives.cpp

```diff
@@ -2087,6 +2089,96 @@ int prArrayContainsSeqColl(struct VMGlobals *g, int numArgsPushed)
 	return errNone;
 }
 
+int sc_arrayMaxDepth(PyrSlot *a, int depth)
+{
+	PyrSlot *slot, *endptr;
+	PyrObject *obj;
+	int size, newdepth;
+	newdepth = depth;
+	obj = slotRawObject(a);
+	size = obj->size;
+	slot = obj->slots - 1;
+	endptr = slot + size;
+	while (slot < endptr) {
```

Comment:
Ok, I'm not changing it here, because I'd have to do it consistently across the file and we need to test the other methods then.